src/Uno.UWP/System/DispatcherQueueTimer.cs

```diff
@@ -92,12 +123,12 @@ public void Stop()
 		 */
 		private void Restart(TimeSpan interval)
 		{
-			var now = DateTimeOffset.Now;
+			StartTimestamp = DateTimeOffset.UtcNow;
```

Comment:
Shouldn't we use a stopwatch instead (for precision) ?

Reply:
We could. The idea here was only to publicly (actually internally) expose the existing info instead of re-build it for each user, especially since accessing to `DateTimeOffset.[Utc]Now` is known to be quite expensive. In any case the precision of the clock is sufficient enough for inertia computation.